src/helpers/charts/gauge_chart.ts

```diff
@@ -179,16 +181,23 @@ export class GaugeChart extends AbstractChart {
 
   static transformDefinition(
     definition: GaugeChartDefinition,
+    chartSheetId: UID,
+    sheetMap: Record<string, UID>,
     executed: AddColumnsRowsCommand | RemoveColumnsRowsCommand
   ): GaugeChartDefinition {
     let dataRangeZone: UnboundedZone | undefined;
+    let sheetName: string | undefined;
```

Comment:
```suggestion
    let newDataRange: string | undefined;
```
nitpick, that way we don't have to have two `let` variables hanging around

same in scorecard